### ISSUE TITLE
Question: how to extract value from StringValue?

### ISSUE THREAD

**Issue by MaceWindu** [AngleSharp/AngleSharp.Css#26]
I need to access actual value of CSS string literal (`StringValue` class). Right now I can only access CssText property, which is string literal itself with quotes and required escaping.

Basically I would like to access `StringValue.Value` property, but it is not public right now. Maybe you can add ICssStringValue interface with additional Value property?

**Comment by FlorianRappl** [comment]
See #24 for info on the CSSOM in general.

I am not sure if makes sense to leave multiple issues open for this one, but since this particular issue is called "Question" I would close it and keep only #24 open. HTH!

**Comment by MaceWindu** [comment]
Want to reopen it as I still cannot access `Value` property without reflection with 0.12.1. I'm getting [`Label`](https://github.com/AngleSharp/AngleSharp.Css/blob/master/src/AngleSharp.Css/Values/Primitives/Label.cs#L9) instance, which available only through [ICssPrimitiveValue](https://github.com/AngleSharp/AngleSharp.Css/blob/master/src/AngleSharp.Css/Values/ICssPrimitiveValue.cs) interface, which doesn't expose `Value` property

**Comment by FlorianRappl** [comment]
With v0.12 we made some progress here, but the big part of CSSOM is still not done (see the related story for it).

This issue has been closed as the question / the real one being #6, however, we could create an independent feature issue for the upcoming v0.13 to track the change on values alone. The feature could be labeled "Draft for values" as in v0.13 the CSSOM would not be closed and values represent a huge chunk of the CSSOM.
